src/test/java/com/flightstats/hub/metrics/StatsdReporterIntegrationTest.java

```diff
@@ -43,26 +56,43 @@ private static StatsdReporter provideStatsDHandlers() {
         return provider.get();
     }
 
-    private static void runAfterSocketInit(Runnable runnable) {
-        Executors.newScheduledThreadPool(1).schedule(runnable, 100, TimeUnit.MILLISECONDS);
-    }
-
-
-    @BeforeClass
-    public static void startMockStatsDServer() {
+    public void startMockStatsDServer() throws InterruptedException, ExecutionException, TimeoutException {
```

Comment:
InterruptedException is already caught in catch block. We don't need this throw. We should just use lombok's sneakyThrows annotation

Reply:
I'm cool with using a @SneakyThrows here since it's in a test, but if I understand how the annotation works, I think we need to be careful about where we use it. Not using checked exceptions feels like it obfuscates the API of the method.

Reply:
FWIW, my SneakyThrows usage typically falls into two cases:

* I would have just wrapped it in a RuntimeException anyway.
* I'm only adding it to the method declaration because I have no idea what to do with it. Now, if an upper level can make a smart decision with the checked exception, you want to explicitly have it on the method throws declaration. If nothing in your code base is going to intelligently handle the checked exception, then you might as well SneakyThrows.

Reply:
There have definitely been some points in hub code where the latter point applied. It felt weird.